src/app/apply/page.tsx

```diff
@@ -83,15 +85,21 @@ export default function Page() {
       });
 
       setIsApplied(true);
-
-      // if (resData.message === "Applied Successfully") reset()
     } catch (err: any) {
       toast({
         title: `Error`,
         description: `${err.message}`,
         status: "error",
         isClosable: true,
       });
+
+      if (err.message == "This Email is already Occupied!") {
```

Comment:
These message checks need to be updated based on the updated text I've suggested.